PROCESS.md

```diff
@@ -0,0 +1,200 @@
+
+If an existing schema under the new release exhibits defined behavior that is contrary to defined behavior under the previous release, the new release is said to contain breaking changes and the specification is unstable between those releases.
+
+If a new release fully defines a previously undefined (or under-defined) behavior, the new release is still considered compatible, even if it contradicts the decision of any particular implementation.
+
+For reference, this table shows the validation results of a hypothetical schema and instance across two consecutive releases to illustrate the compatibility of those releases:
+
+<table>
+  <tr>
+    <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Next➡️<br>⬇️Current</td>
+    <th width="150" align="center">pass</th>
+    <th width="150" align="center">fail</th> 
+    <th width="150" align="center">indeterminant</th> 
+  </tr>
+  <tr>
+    <th align="center">pass</th>
+    <td align="center">✅</td> 
+    <td align="center">❌</td>
+    <td align="center">❌</td>
+  </tr>
+  <tr>
+    <th align="center">fail</th>
+    <td align="center">❌</td> 
+    <td align="center">✅</td>
+    <td align="center">❌</td>
+  </tr>
+  <tr>
+    <th align="center">indeterminant</th>
+    <td align="center">✅</td> 
+    <td align="center">✅</td> 
+    <td align="center">✅</td>
+  </tr>
+</table>
+
+### Release
+
+A release is any single publication of the JSON Schema specifications (as a group).
+
+### Version
+
+Consecutive releases which maintain compatibility with each other comprise a version.
+
+## Release and Version
+
+The JSON Schema specification will aim to publish annually on or about the First of January each year.  Releases are identified by the year they are published.
+
+When a new release contains breaking changes, that release begins a new version of JSON Schema.
+
+The version will be identified as an integer, starting with `1` and incrementing as needed.
+
+Stability will be prioritized when making changes to the specification.  Breaking changes are undesired and should be avoided when possible.
+
+## Publication
+
+### Specifications
+
+The specifications will be published on the JSON Schema website, https://json-schema.org/, using a path comprised of the version, year, and document name.  For example,
+
+```
+https://json-schema.org/1/2025/core.html
+https://json-schema.org/1/2025/validation.html
+```
+
+Once a specification document has been published, neither the document (save for minor errata such as spelling mistakes) nor its publication URL may change.  If the TSC elects to alter the above URL scheme, the new scheme only applies to future publications and are not retroactive.
+
+### Meta-schemas
+
+A release meta-schema will be published under the same path using `schema` as the file name.
```

Comment:
We need a file name.  The release year is a folder, as indicated by the spec document paths.  This just publishes `schema` as a sibling file.

Reply:
The file can be `{release-year}.schema.json` and we can use CF configs to serve that file when `https://json-schema.org/1/2025` is requested. It's a little extra maintenance work, but I think the shorter URI is worth it.

Reply:
I don't think it's worth the trouble.  It's a file name, and `https://json-schema.org/1/2025/schema[.json]` makes it consistent with the spec file names.  Everything is under the year _folder_.

Reply:
> We can still make it the default file for the folder if you want

Yes, that was my suggestion. I don't care too much where it lives or what the file is called, but I think it's important that the meta-schema URI be as simple as possible since it's effectively required to use in every schema. And although it doesn't technically matter, it will cause a lot of confusion if the schema isn't retrievable from the meta-schema URI.

As long as the intention isn't that the meta-schema URI include `schema[.json]`, then I'm happy as long as there's something configured to serve `https://json-schema.org/1/2025/schema.json` on requests to `https://json-schema.org/1/2025`.

Actually, I do care about one aspect of the file name. I think it should be either `schema.json` or `[whatever].schema.json`. Not having an appropriate extension makes it much more difficult to work with when we're editing meta-schemas. Traditionally, it was done that way so we could serve the file at the same URI as the meta-schema URI. Since we moved to cloudflare, we have better ways to deal with that problem and we're not limited to extensionless filenames.

Reply:
Okay.  I'm updating this to use `schema.json` as the file name and to also serve the file from the folder URLs.

We need to have the `$id` conversation in the [discussion thread](https://github.com/orgs/json-schema-org/discussions/671#discussioncomment-8753266).

Reply:
> We need to have the `$id` conversation in the [discussion thread](https://github.com/orgs/json-schema-org/discussions/671#discussioncomment-8753266).

Agreed. To be clear, I originally thought you were proposing that `https://json-schema.org/1/2025/schema` should be the `$id`.